src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/Audit/AuditHelperTests.cs

```diff
@@ -103,6 +124,26 @@ public void GivenNoAuditEventType_WhenLogExecutedIsCalled_ThenAuditLogShouldNotB
                 callerClaims: default);
         }
 
+        [Theory]
```

Comment:
Copy/Paste? This test is identical to the one added above.

Reply:
one is for LogExecuting and other is for LogExecuted